README.md

```diff
@@ -130,6 +130,9 @@ To keep all those hundreds of source files from ending up in the same directory,
 
 First, create them as groups (little yellow "folders") within the group with your project's name in Xcode's Project Navigator. Then, for each of the groups, link them to an actual directory in your project path by opening their File Inspector on the right, hitting the little gray folder icon, and creating a new subfolder with the name of the group in your project directory.
 
+However, in case of a complex project, splitting the files based on the features is a good idea to keep in mind. 
```

Comment:
"However, for complex projects, ..."
"splitting the files by feature is a good idea."
Maybe?